src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/Methodologies/MethodologyAmendmentServiceTests.cs

```diff
@@ -88,6 +88,100 @@ await using(var context = InMemoryApplicationDbContext(contextId))
                 Assert.Equal("Content!", contentBlock.Body);
             }
         }
+        
+        [Fact]
+        public async Task CreateMethodologyAmendmentWithMethodologyFiles()
+        {
+            var originalMethodology = new Methodology
+            {
+                Id = Guid.NewGuid(),
+                Status = Approved,
+                Published = DateTime.Today,
+                MethodologyParent = new MethodologyParent
+                {
+                    Slug = "methodology-slug",
+                    OwningPublicationTitle = "Owning Publication Title"
+                },
+                Content = AsList(new ContentSection
+                {
+                    Content = AsList<ContentBlock>(new HtmlBlock
+                    {
+                        Body = "Content!"
+                    })
+                })
+            };
+
+            var originalMethodologyFile1 = new MethodologyFile
+            {
+                Id = Guid.NewGuid(),
+                MethodologyId = originalMethodology.Id,
+                File = new File
+                {
+                    Id = Guid.NewGuid()
+                }
+            };
+
+            var originalMethodologyFile2 = new MethodologyFile
+            {
+                Id = Guid.NewGuid(),
+                MethodologyId = originalMethodology.Id,
+                File = new File
+                {
+                    Id = Guid.NewGuid()
+                }
+            };
+            
+            var contextId = Guid.NewGuid().ToString();
+            await using(var context = InMemoryApplicationDbContext(contextId))
+            {
+                await context.Methodologies.AddAsync(originalMethodology);
+                await context.MethodologyFiles.AddRangeAsync(originalMethodologyFile1, originalMethodologyFile2);
+                await context.SaveChangesAsync();
+            }
+
+            Guid amendmentId;
+
+            // Call the method under test
+            await using(var context = InMemoryApplicationDbContext(contextId))
+            {
+                var methodologyService = new Mock<IMethodologyService>();
```

Comment:
I see if this wasn't used the capture wouldn't happen and later checks would fail but maybe this could be a strict mock and still verify all mocks at the end?

Reply:
Yup good idea - I make them Strict on autopilot most of the time - don't know why I didn't here!